myia_frontend_pytorch/pyproject.toml

```
@@ -0,0 +1,33 @@
+[build-system]
+requires = ["poetry>=0.12"]
+build-backend = "poetry.masonry.api"
+
+[tool.poetry]
+name = "myia_frontend_pytorch"
+version = "0.1.0"
+description = "Pytorch frontend for myia"
+authors = ["Mila development team"]
+
+[tool.poetry.dependencies]
+python = "^3.7"
+numpy = "^1.16"
+torch = "1.5.0"
+torchvision = "0.6.0"
+
+[tool.poetry2conda]
+name = "myia_frontend_pytorch"
+
+[tool.poetry2conda.dependencies]
+torch = { channel = "pytorch", name = "pytorch" }
+torchvision = { channel = "pytorch" }
+
+[tool.isort]
+known_first_party = "myia_backend_pytorch"
```

```suggestion
known_first_party = "myia_frontend_pytorch"
```

Also, I don't think this section should be here unless `isort` is in the dev requirements for this package.

Thinking about it, the same comment applies to the `pyproject.toml` of the backends.

IMO, while these packages are in subdirectories of the main one, the `isort` section should only be specified for the main `myia` package.